v3/opt/ecr/aws-ecr-login.service

```diff
@@ -7,6 +7,7 @@ After=network-online.target
 
 [Service]
 Type=oneshot
+EnvironmentFile=/etc/environment
```

Comment:
That file was written as part of the cloud-config:
https://github.com/adobe-platform/infrastructure/tree/master/v2/src/userdata/templates/write_files/environment
Specifically:
https://github.com/adobe-platform/infrastructure/blob/master/v2/src/userdata/templates/write_files/environment/worker.yml#L2